### PR TITLE
Fix examples to build cleanly if PICO_DEFAULT_LED_PIN isn't defined

### PR DESCRIPTION
**Pull request by lurch**

### CODE REVIEW

**Comment by kilograham** [comment]
hmm i could swear i have a PR for this too!

**Comment by kilograham** [comment]
note i will combine this with my changes - the thing I did differently for sure was to make the double_tap work without a LED if there isn't one, as the LED is not very important to the functionality of the example

**Comment by lurch** [comment]
Feel free to chop'n'change or pick'n'mix my PRs however you see best :+1: 

> make the double_tap work without a LED if there isn't one, as the LED is not very important to the functionality of the example

That's a good point, in my naive copy'n'pasting I hadn't considered that.

File: system/hello_double_tap/hello_double_tap.c [hunk 1]
```diff
@@ -13,6 +13,9 @@
 // `pico_bootsel_via_double_reset` library!
 
 int main() {
+#ifndef PICO_DEFAULT_LED_PIN
```
Comment (kilograham) [inline system/hello_double_tap/hello_double_tap.c:16]: yup, i seem to have lost my changes for this... can you fix this one up to just not use the LED instead

**Review by kilograham — CHANGES_REQUESTED** [review]

File: system/hello_double_tap/hello_double_tap.c (hunk 1 above)
Comment (lurch) [reply system/hello_double_tap/hello_double_tap.c]: Done.

**Review by kilograham — APPROVED** [review]

**Pull request merged**